spec/integration/make/dependencies_spec.rb

```diff
@@ -1,10 +1,13 @@
 require "spec_helper"
 
 RSpec.describe "Make dependencies" do
+  # Some dependencies are only needed by the dependant app for their static files.
+  dependencies_that_do_not_need_to_be_running = %w[govuk-content-schemas]
+
   ProjectsHelper.all_projects.each do |project_name|
     it "mirrors docker-compose.yml for #{project_name}" do
-      expect(app_dependencies_in_compose_file(project_name))
-        .to match_array(app_dependencies_in_makefile(project_name))
+      expect(app_dependencies_in_compose_file(project_name) - dependencies_that_do_not_need_to_be_running)
```

Comment:
Agree with subtracting in both places: this stops someone accidentally adding it as a dependency in their Compose file.

Minor: could we make this part of the methods instead? One of them already does a subtraction. "app" is a vague enough term that the methods should still make sense.

Reply:
Good idea 👍 I did try this initially but had some issues, I think because I was wrapping the variable in `[]` 🤦 I've pushed up a commit for this, and a new test for the Makefiles.